.azure-pipelines.yml

```diff
@@ -0,0 +1,298 @@
+# See https://aka.ms/yaml for documentation
+
+trigger:
+  branches:
+    include:
+    - '*'
+  tags:
+    include:
+    - '*'
+
+variables:
+  python.version: '3.7'
```

Comment:
Bumping to 3.7? Does snap support that?

Reply:
snap build with 3.6, so we should stick to 3.6 !